ckanext/grouphierarchy/templates/group/snippets/output_data_list.html

```diff
@@ -0,0 +1,19 @@
+{#
+Displays a list of datasets.
```

Comment:
This is heavily inspired by the implementation [in `ckanext-nextgeoss`](https://github.com/NextGeoss/ckanext-nextgeoss/blob/master/ckanext/nextgeoss/templates/group/snippets/output_data_list.html#L18). It's just that [this check](https://github.com/NextGeoss/ckanext-nextgeoss/blob/master/ckanext/nextgeoss/helpers.py#L541) was failing for these datasets becase the value is actually "true" (as in JSON). At the moment I can't tell why that JSON value is not converted and since we're in a rush I thought I'd just deal with it in this plugin especially since we will have to refactor the groups in `ckanext-nextgeoss` soon.